src/Evolution/Systems/GeneralizedHarmonic/Initialize.hpp

```diff
@@ -46,7 +45,7 @@ namespace GeneralizedHarmonic {
 namespace Actions {
 template <size_t Dim>
 struct InitializeConstraintsTags {
-  using initialization_option_tags = tmpl::list<>;
+  using initialization_tags = tmpl::list<>;
```

Comment:
delete these empty lists. They are empty by default.

Reply:
Yes - but I have a mild preference for keeping it explicit for aesthetic completeness and easier readibility for newcomers, unless there's actually a runtime performance  hit.